doc/install_mne_python.rst

```diff
@@ -29,89 +21,74 @@ Step-by-step instructions to accomplish this are given below.
-environment that resolves all dependencies, we highly recommend following
-installation instructions for the Anaconda Python distribution. Go here
-to download, and follow the installation instructions:
-
-* https://www.continuum.io/downloads
-* http://docs.continuum.io/anaconda/install
+environment that resolves all dependencies, **we recommend the
+Anaconda Python distribution**.
 
-.. note:: Intel's `Math Kernel Library <https://software.intel.com/en-us/intel-mkl>`_
-          speeds up many linear algebra comptutations, some by an order of
-          magnitude. It is included by default in Anaconda, which makes it
-          preferable to some other potentially viable Python interpreter
-          options (e.g., using ``brew`` in OSX or ``sudo apt-get install``
-          on Ubuntu to get a usable Python interpreter). 
+Python has two major versions currently available, 2.7+ and 3.3+. Currently
+the 3D visualization dependencies `Mayavi`_ and `PySurfer`_ only run out of the
+box on Python 2.7, so **we recommend using Python 2.7**. You can get
+Anaconda 2.7 for Windows, OSX, and Linux download and installation instructions
+`from the ContinuumIO site <http://docs.continuum.io/anaconda/install>`_.
 
-If everything is set up correctly, you should be able to check the version
-of ``conda`` that is installed (your version number will probably be newer)
-and which ``python`` will be called when you run ``python``:
+Once everything is set up, you should be able to check the version
+of ``conda`` and ``python`` that is installed:
 
 .. code-block:: bash
 
     $ conda --version
-    conda 3.19.1
+    conda 4.2.14
     $ which python
     /home/agramfort/anaconda/bin/python
+    $ python --version
+    Python 2.7.12 :: Continuum Analytics, Inc.
 
-If your installation doesn't look something like this,
-*something went wrong* and you should try to fix it. We recommend looking
-through the Anaconda documentation a bit, and Googling for Anaconda
-install tips (StackExchange results are often helpful).
-
-Once Anaconda works properly, you can do this to resolve
-the MNE-Python dependencies:
-
-.. code-block:: bash
-
-    $ conda install scipy matplotlib scikit-learn
+.. note:: If your installation doesn't look something like this,
+          *something went wrong* and you should try to fix it. Try looking
+          through the Anaconda documentation or Googling for Anaconda
+          install tips (StackExchange results are often helpful).
 
-To test that everything works properly, open up IPython:
+You can then do this to resolve the MNE dependencies:
 
 .. code-block:: bash
 
-    $ ipython --matplotlib=qt
-
-Now that you have a working Python environment you can install MNE-Python.
-
-If you want to have an environment with a clean MATLAB-like interface,
-consider using Spyder_, which can easily be installed with Anaconda
-as:
+    $ conda install scipy matplotlib scikit-learn mayavi
```

Comment:
Should we add `ipython`? When I did this on a new env, iPython was not installed, and hence `$ ipython` opened my root environment which was confusing for a moment.